File: java/Cup_Cakes/SmallestNumber.java
public class SmallestNumber{

public static void main(String []args){

int[] number = {10, 9, 13, 2, 14};

System.out.println(smallest(number));


}


public static int smallest(int[] arrNumber){
int smallest = arrNumber[0];

for(int index = 0; index < arrNumber.length; index++){
	if(arrNumber[index] < smallest){
	smallest = arrNumber[index];
	
	}
	
}

	return smallest;

}

}

File: javaScript/Task/smallestNumber.js

let number = 123459;
let smallest = 1;

let num = number.toString();

for(let index = 1; index < num.length; index++){
	
	let output = parseInt(num[index]);
	
	if(output < smallest){
	smallest= output
	}
	
}

console.log(smallest);